src/main/groovy/org/mockito/release/internal/gradle/util/PomCustomizer.groovy

```diff
@@ -82,13 +85,14 @@ class PomCustomizer {
                 d.appendNode('url', "https://github.com/${split[0]}")
             }
 
-            def contributors = root.appendNode('contributors')
-            rootProject.ext.pom_contributors.each {
-                def split = it.split(':')
-                assert split.length == 2
-                def c = contributors.appendNode('contributor')
-                c.appendNode('name', split[1])
-                c.appendNode('url', "https://github.com/${split[0]}")
+            def contributorsNode = root.appendNode('contributors')
+            def fileName = Contributors.getAllProjectContributorsFileName(project.getBuildDir())
+            AllProjectContributorsReader reader = AllProjectsContributorsProvider.allProjectContributorsReader()
```

Comment:
Instead of reading the file on the fly here, every time we generate the pom, I suggest an alternative approach:

1. We add the fetcher task that fetches from GitHub and write the file. This task is incremental and does not fetch from github if file was already fetched. The task should be on root project only (continuous delivery plugin?) because we only want to fetch once for the entire project, regardless of how many jars we produce.
2. We add 'teamConfigurer' task (depends on fetcher) that loads the file and configures the releasing.team object (needs my PR to be merged first https://github.com/mockito/mockito-release-tools/pull/77). This task is not incremental because it configures internal state (releasing.team object). For the same reasons as previous task, this task should be on the root project, single task for the entire build.
3. We make the every 'generatePom' task depend on teamConfigurer. We can do it continuous delivery plugin, using 'allprojects' and 'withType' trick

The added benefit of doing it this way is that users can easily disable the tasks and explicitly configure developers and contributors on the 'releasing.team' extension.

What do you think?